### ISSUE TITLE
Cannot pair PIR NAS-PD01Z

### ISSUE THREAD

**Issue by Tuscany58** [mruiter/com.neo#215]
Hello 
Some month ago I purchased a few sensors and they worked very well.
Now I got 2 additional PIR sensors and they get not recognized, and are paired as generic z-wave device

Vendor informed me that it is model NAS-PD01Z
Sticker on the box of the motion sensor says
"Z PIR + TEMP 868.4MHZ EU"
Another sticker shows the z-wave plus symbol

Please let me know which information you need to help me get use of theses sensors.
Thank you very much
Peter

**Comment by mruiter** [comment]
Need the device product , manufacturer and producttypeid to add to de app when added as generic device

**Comment by Tuscany58** [comment]
Hello Marco !

Is that what You need? If not let me know how to get information you need.
Thank You in advance

[Ein Bild, das Text enthält.  Automatisch generierte Beschreibung]

[mm_logo2015_800]
-------------------------------------------------------------------------------------
manege.media ● Inh. Peter Lunacek
An der Sonnleithen 15/406 ● A-4020 Linz
Telefon: +43 660 321 0000 ● Email: ***@***.******@***.***>
Internet: www.manege.at<http://www.manege.at/> ● www.mediaseven.at<http://www.mediaseven.at>
-------------------------------------------------------------------------------------

Von: Marco Ruiter ***@***.***>
Gesendet: Donnerstag, 20. Oktober 2022 08:18
An: mruiter/com.neo ***@***.***>
Cc: Tuscany58 ***@***.***>; Author ***@***.***>
Betreff: Re: [mruiter/com.neo] Cannot pair PIR NAS-PD01Z (Issue #215)


Need the device product , manufacturer and producttypeid to add to de app when added as generic device

**Comment by mruiter** [comment]
I dont see any image attached only the text [Ein Bild, das Text enthält.  Automatisch generierte Beschreibung]

**Comment by Tuscany58** [comment]
I hope it is attached now.


[mm_logo2015_800]
-------------------------------------------------------------------------------------
manege.media ● Inh. Peter Lunacek
An der Sonnleithen 15/406 ● A-4020 Linz
Telefon: +43 660 321 0000 ● Email: ***@***.******@***.***>
Internet: www.manege.at<http://www.manege.at/> ● www.mediaseven.at<http://www.mediaseven.at>
-------------------------------------------------------------------------------------

Von: Marco Ruiter ***@***.***>
Gesendet: Donnerstag, 20. Oktober 2022 08:30
An: mruiter/com.neo ***@***.***>
Cc: Tuscany58 ***@***.***>; Author ***@***.***>
Betreff: Re: [mruiter/com.neo] Cannot pair PIR NAS-PD01Z (Issue #215)


I dont see any image attached only the text [Ein Bild, das Text enthält. Automatisch generierte Beschreibung]

**Comment by mruiter** [comment]
Nope sorry.  You can mail it to mruiter@tuxweb.nl

**Comment by Tuscany58** [comment]
OK. Sent it to Your Email.
Thank You very much

Peter

[mm_logo2015_800]
-------------------------------------------------------------------------------------
manege.media ● Inh. Peter Lunacek
An der Sonnleithen 15/406 ● A-4020 Linz
Telefon: +43 660 321 0000 ● Email: ***@***.******@***.***>
Internet: www.manege.at<http://www.manege.at/> ● www.mediaseven.at<http://www.mediaseven.at>
-------------------------------------------------------------------------------------

Von: Marco Ruiter ***@***.***>
Gesendet: Donnerstag, 20. Oktober 2022 08:50
An: mruiter/com.neo ***@***.***>
Cc: Tuscany58 ***@***.***>; Author ***@***.***>
Betreff: Re: [mruiter/com.neo] Cannot pair PIR NAS-PD01Z (Issue #215)


Nope sorry. You can mail it to ***@***.******@***.***>

**Comment by mruiter** [comment]
You can try the new test version

https://homey.app/nl-nl/app/com.neo/NEO-Coolcam/test/



Tuscany58 schreef op 2022-10-20 08:52:

**Comment by Tuscany58** [comment]
Hello Marco !

Thank You very much for the update.
Device now can be connected to homey as a PIR (not a generic device)
I added it as a PIR V2

Unfortunately it takes 5-6 seconds that sensor reports movement to homey, which is too long for switching on lights.

I guess this has nothing to do with your app but is a hardware issue.

[mm_logo2015_800]
-------------------------------------------------------------------------------------
manege.media ● Inh. Peter Lunacek
An der Sonnleithen 15/406 ● A-4020 Linz
Telefon: +43 660 321 0000 ● Email: ***@***.******@***.***>
Internet: www.manege.at<http://www.manege.at/> ● www.mediaseven.at<http://www.mediaseven.at>
-------------------------------------------------------------------------------------

Von: Marco Ruiter ***@***.***>
Gesendet: Donnerstag, 20. Oktober 2022 09:33
An: mruiter/com.neo ***@***.***>
Cc: Tuscany58 ***@***.***>; Author ***@***.***>
Betreff: Re: [mruiter/com.neo] Cannot pair PIR NAS-PD01Z (Issue #215)

You can try the new test version

https://homey.app/nl-nl/app/com.neo/NEO-Coolcam/test/



Tuscany58 schreef op 2022-10-20 08:52:

**Comment by mruiter** [comment]
Yes. This is not something I can solve in an app . It either reports movement or not . The time it takes is in the hardware / firmware of the device Verstuurd vanaf mijn iPhoneOp 20 okt. 2022 om 10:38 heeft Tuscany58 ***@***.***> het volgende geschreven:﻿
Hello Marco !



Thank You very much for the update.

Device now can be connected to homey as a PIR (not a generic device)

I added it as a PIR V2



Unfortunately it takes 5-6 seconds that sensor reports movement to homey, which is too long for switching on lights.



I guess this has nothing to do with your app but is a hardware issue.



[mm_logo2015_800]

-------------------------------------------------------------------------------------

manege.media ● Inh. Peter Lunacek

An der Sonnleithen 15/406 ● A-4020 Linz

Telefon: +43 660 321 0000 ● Email: ***@***.******@***.***>

Internet: www.manege.at<http://www.manege.at/> ● www.mediaseven.at<http://www.mediaseven.at>

-------------------------------------------------------------------------------------



Von: Marco Ruiter ***@***.***>

Gesendet: Donnerstag, 20. Oktober 2022 09:33

An: mruiter/com.neo ***@***.***>

Cc: Tuscany58 ***@***.***>; Author ***@***.***>

Betreff: Re: [mruiter/com.neo] Cannot pair PIR NAS-PD01Z (Issue #215)



You can try the new test version



https://homey.app/nl-nl/app/com.neo/NEO-Coolcam/test/







Tuscany58 schreef op 2022-10-20 08:52:

**Comment by CliffBroeren** [comment]
I Have the Same issue with the NAS-PD01  
I have know installed the expirimental 3.0.7 version app

Then i can Install the Sensors 
But there is no temp reading 

on the box of the sensor they said + TEMP
